doc/getting-started/apl-nuc.rst

```diff
@@ -281,6 +281,18 @@ partition. Follow these steps:
 #. From here you can login as root using the password you set previously when
    you installed Clear Linux.
 
+#. (**Optional**) The ``software-defined-cockpit`` bundle installs the
+   ``ioc-cbc-tools`` on the system and activates three ``systemd`` services.
+   Those services do not work on off-the-shelf platforms such as NUCs, UP2.
```

Comment:
Seems wrong to have software installed and services run that aren't supported on the open source defined hardware platforms.

Reply:
It is wrong but until we get these service files fixed, this is my best recommendation.

Reply:
Agree, blindly running cbc service and waiting for service to die (30 seconds later) is not good solution. But there isn't really good way to detect IOC chip, unless ACPI table could give us some hints.

Reply:
Is there any entry from the ACPI table we could use for that, something like a PlatformID or anything else? (disclaimer: I really don't know much about ACPI tables)